.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4" # Optional: defaults to "gpt-4"
+          OPENAI_API_MODEL: "gpt-3" # Optional: defaults to "gpt-4"
```

Comment:
I think you should use GPT-4. It's much better to review code than GPT-3. It's more expensive but I don't think it will have too much impact. It becomes costly when you have to parse multiple documents. The AI bot only reviews the code being changed (the diff)

Reply:
It didn't work, I don't know why, I use gpt4

Reply:
It seems that don't take my secret from organization, the api key seems never used

Reply:
If it still doesn't work, try with gpt-4-0613
Or maybe you need to remove the quotes